lesson_10/solid/solid_app/src/test/java/com/codedifferently/lesson10/library/LibraryTest.java

```diff
@@ -147,4 +160,22 @@ void testLibrary_preventsRemovingCheckedOutBooks() {
         .isInstanceOf(BookCheckedOutException.class)
         .hasMessage("Cannot remove checked out book.");
   }
-}
+
+  @Test
+  void testLibrary_allowLibrarianToCheckout() {
```

Comment:
The requirement is that librarians are needed to add and remove items from the library, not check in and out.